mm2src/coins/z_coin/z_coin_errors.rs

```diff
@@ -170,6 +171,7 @@ impl From<GenTxError> for WithdrawError {
             GenTxError::DecryptedOutputNotFound
             | GenTxError::FailedToGetMerklePath
             | GenTxError::PrevTxNotConfirmed
+            | GenTxError::NeededPrevTxConfirmed
```

Comment:
This sounds more like a [timeout](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) rather than internal error.